Makefile

```diff
@@ -11,6 +11,19 @@ $(MBEDTLS_PATH)/framework/exported.make not found.
 Run `git submodule update --init` to fetch the submodule contents.
 This is a fatal error
         endef
+        ifeq (,$(wildcard .git))
```

Comment:
I think the definition above (lines 9-13) is redundant with the one on lines 21-25. I'd keep only the second one.